### DISCUSSION TITLE
1.2.1 release

### DISCUSSION THREAD

**Discussion by evpobr** [libsndfile/libsndfile#951] [General]
@arthurt, @SoapGentoo , @erikd , planning v1.2.1 bugfix release before WavPack. If no one is against of course.

**Comment by SoapGentoo** [comment]
go for it, but please prepare everything so I just have to tag and create the tarball

**Reply by evpobr** [reply]
https://github.com/libsndfile/libsndfile/tree/release-1.2.1

Just set date in `CHANGELOG.md` & `docs/index.md`.

**Reply by evpobr** [reply]
@SoapGentoo? Are you very busy?

**Reply by SoapGentoo** [reply]
@evpobr sorry, I wasn't aware that everything is ready

**Reply by SoapGentoo** [reply]
can you please finalise so I just have to bootstrap the tarball

**Reply by evpobr** [reply]
@SoapGentoo , completed.

**Reply by SoapGentoo** [reply]
ok starting now

**Reply by SoapGentoo** [reply]
ok, I've found two more small bugs, one in C code and one in python, how do you want to proceed?

**Reply by SoapGentoo** [reply]
```
../../tests/string_test.c: In function ‘string_start_test’:
../../tests/string_test.c:447:54: error: ‘%s’ directive argument is null [-Werror=format-overflow=]
  447 |                         printf ("    Bad software  : %s\n", cptr) ;
      |                                                      ^~
../../tests/string_test.c:447:54: error: ‘%s’ directive argument is null [-Werror=format-overflow=]
cc1: all warnings being treated as errors
```

**Reply by evpobr** [reply]
Already in master :smile:

**Reply by SoapGentoo** [reply]
I suggest
- unprotecting master
- removing the 1.2.1 commit
- adding my two fixes
- and readding the release commit

since this is still very fresh

**Reply by evpobr** [reply]
Sorry for releasing too fast. Rewriting remote history is not good IMHO. Maybe leave it as it is and release 1.2.2 with your fixes tomorrow?

**Reply by SoapGentoo** [reply]
ok then 1.2.2 tomorrow

**Reply by evpobr** [reply]
Tag 1.2.1 please, nothing bad with it, except missing two fixes. Windows ZIPs are ready, I will upload it tomorrow.

**Reply by SoapGentoo** [reply]
and then do 1.2.2 tomorrow?

**Reply by evpobr** [reply]
Yes

**Comment by arthurt** [comment]
Sounds good.

**Reply by evpobr** [reply]
[libsndfile-1.2.1-win32.zip](https://github.com/libsndfile/libsndfile/files/12328771/libsndfile-1.2.1-win32.zip)
[libsndfile-1.2.1-win64.zip](https://github.com/libsndfile/libsndfile/files/12328772/libsndfile-1.2.1-win64.zip)

**Comment by arthurt** [comment]
Maybe we can get #930 in too? It fell-off my view.

**Reply by evpobr** [reply]
Good idea.

**Reply by evpobr** [reply]
I guess you need to make signed commit, not only signed tag. Correct?

**Reply by SoapGentoo** [reply]
please then update the dates, I will not do anything beyond signing and tagging
